export { catchExceptionFactory } from './catch-exception.factory';
